src/backend/api/usuario/usuario.schema.js

```diff
@@ -24,9 +24,9 @@ module.exports = {
         type: 'number',
         description: 'reputacao do usuário',
       },
-      profilePicture: {
+      fotoPerfil: {
         type: 'string',
```

Comment:
acho melhor usar o objectId do mongo para fazer referencia a imagem

Reply:
sim, futuramente eu pretendo fazer isso, pois ainda é nescessário criar uma logica de acesso as imagens no banco de dados melhor, quando isso for feito isso daí vai ser o Id